console/billing/concepts.mdx

```diff
@@ -63,9 +63,9 @@ IAM billing permissions can be given to users to allow them to read and/or manag
 
 **K**now **Y**our **C**ustomer is the process of verifying a client's identity. An ID document such as a passport, driver's license or identity card is required to [complete KYC verification](/console/account/how-to/verify-identity/).
 
-## Pay as you go
+## Pay-as-you-go
 
-Scaleway provides a comprehensive suite of public cloud services designed around the **pay as you go** model. This guarantees cost efficiency by charging you solely for the resources you consume. Among our offers are **on-demand services**, which operate on a pay as you go basis, granting you immediate access to resources at a fixed rate per hour.
+Scaleway provides a comprehensive suite of public cloud services designed around the **pay-as-you-go** model. This guarantees cost efficiency by charging you solely for the resources you consume. Among our offers are **on-demand services**, which operate on a pay as you go basis, granting you immediate access to resources at a fixed rate per hour.
```

Comment:
```suggestion
Pay-as-you-go is a billing system in which you only pay for what you consume instead of having a pre-defined fixed amount. Scaleway offers a comprehensive suite of public cloud services that operate this way so you can benefit from cost-efficient usage. Our **on-demand** services operate on a pay-as-you-go basis, granting you immediate access to resources at a fixed hourly rate.
```

Reply:
Just a suggestion, so the definition of the concept is the first thing that appears on the sentence.